src/extensions/agent_modules/cloud_communication/eg_mqtt_broker_client/tests/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required (VERSION 3.5)
+
+project (adu_mqtt_client_module_functional_test)
+
+include (agentRules)
+
+compileasc99 ()
+disablertti ()
+
+set (
+    sources
+    main.cpp
+    adu_mqtt_client_module_ft.cpp
+    )
+
+find_package (Catch2 REQUIRED)
+
+add_executable (${PROJECT_NAME} ${sources})
+
+target_include_directories (
+    ${PROJECT_NAME}
+    PRIVATE ${ADUC_EXPORT_INCLUDES}
+            ${ADU_EXTENSION_INCLUDES}
+            ${ADU_AGENT_SDK_INCLUDES}
+            ${PROJECT_SOURCE_DIR}/inc
+            ${PROJECT_SOURCE_DIR}/../inc
+            ${PROJECT_SOURCE_DIR}/../mosquitto_client_extensions/inc
+    )
+
+target_compile_definitions (
+    ${PROJECT_NAME}
+    PRIVATE ADUC_TEST_DATA_FOLDER="${ADUC_TEST_DATA_FOLDER}")
+
+target_link_libraries (
+    ${PROJECT_NAME}
+    PRIVATE aduc::config_utils
+            aduc::microsoft_du_mqttbroker_client;
+            aduc::parser_utils
+            aduc::system_utils
+            aduc::mosquitto_client_extension_c
+            Catch2::Catch2)
```

Comment:
again, FT I'd argue should be standalone app not involved with Catch2, CTests, unit tests.

Reply:
These are now standalone apps.